Makefile

```diff
@@ -1,43 +1,26 @@
-CONFIGUREFLAGS=--enable-tests
+.PHONY: release build install uninstall clean test doc reindent
 
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
+release:
+	jbuilder build @install
 
-SETUP = ocaml setup.ml
+build:
+	jbuilder build @install --dev
 
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+install:
+	cp ./_build/default/src/squeezed.exe ./squeezed.native
 
-doc: setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
-
-test: setup.data build
-	$(SETUP) -test $(TESTFLAGS)
-
-all:
-	$(SETUP) -all $(ALLFLAGS)
-
-install: setup.data
-	$(SETUP) -install $(INSTALLFLAGS)
-
-uninstall: setup.data
-	$(SETUP) -uninstall $(UNINSTALLFLAGS)
-
-reinstall: setup.data
-	$(SETUP) -reinstall $(REINSTALLFLAGS)
+uninstall:
+	rm ./squeezed.native
 
 clean:
-	$(SETUP) -clean $(CLEANFLAGS)
-
-distclean:
-	$(SETUP) -distclean $(DISTCLEANFLAGS)
-
-setup.data:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
+	jbuilder clean
 
-configure:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
+test:
+	jbuilder runtest
 
-.PHONY: build doc test all install uninstall reinstall clean distclean configure
+# requires odoc
+doc:
+	jbuilder build @doc
 
-# OASIS_STOP
+reindent:
+	ocp-indent --inplace **/*.ml*
```

Comment:
Use
```
git ls-files '*.ml*' | xargs ocp-indent --inplace
```
it's more robust (I keep forgetting about it, but since @edwintorok thought about it, we decided to stick with it)